build.gradle

```diff
@@ -165,6 +165,9 @@ dependencies {
     api group: 'com.google.guava', name: 'guava', version:'30.0-jre'
     api group: 'commons-lang', name: 'commons-lang', version: '2.6'
     testFixturesImplementation "org.opensearch.test:framework:${opensearch_version}"
+    testImplementation group: 'net.bytebuddy', name: 'byte-buddy', version: '1.12.22'
+    testImplementation group: 'org.objenesis', name: 'objenesis', version: '3.2'
+    testImplementation group: 'net.bytebuddy', name: 'byte-buddy-agent', version: '1.12.22'
```

Comment:
this is required to mock static classes/methods

Reply:
Quick feedback on this:  
1. Is there a possibility that we fix src so that we don't need to mock static functions.
2. Are we using junit4 or junit5 here?

Reply:
Making changes in source will be problematic, there are many such classes involved in call chain.
We're using junit4, 4.13 to be precise.